app/models/ability.rb

```diff
@@ -0,0 +1,14 @@
+class Ability
+  include CanCan::Ability
+  # if operator ||=
+  def initialize(user)
+    user ||= User.new
+    if user.admin?
+      can :manage, :all?
```

Comment:
- [ ] The reason you have the access denied errors in those pages is because you should have `all` (without ? mark), not `all?`